manager/manager.go

```diff
@@ -47,6 +47,15 @@ func (m *Manager) Refresh(environmentId string, update bool) error {
 	return m.refreshEnvironmentCatalogs(environmentId, update)
 }
 
+func (m *Manager) RefreshOne(environmentId string, catalogName string, update bool) error {
```

Comment:
If I understand this function correctly, it is going to reload from disk a catalog on every single GET request. If thats the case, I'm not sure thiat this is ok/will work.